src/ess/isissans/data.py

```diff
@@ -94,22 +71,54 @@ class IsisRegistry:
 def get_path(filename: FilenameType) -> FilePath[FilenameType]:
     """Translate any filename to a path to the file obtained from pooch registries."""
     for reg in _registries:
-        filename = reg.mapping.get(filename, filename)
-        if filename in reg.registry:
+        if filename in reg:
             if filename.endswith('.zip'):
-                return reg.registry.get_path(filename, unzip=True)[0]
-            return reg.registry.get_path(filename)
+                return reg.get_path(filename, unzip=True)[0]
+            return reg.get_path(filename)
+
+
+def get_sans2d_tutorial_data_folder() -> DataFolder:
+    """Get the path to the folder containing the SANS2D tutorial data."""
+    for filename in (
+        'DIRECT_SANS2D_REAR_34327_4m_8mm_16Feb16.dat.h5',
+        'SANS2D00063091.nxs.h5',
+        'SANS2D00063114.nxs.h5',
+        'SANS2D00063159.nxs.h5',
+    ):
+        path = get_path(filename)
+    return DataFolder(str(Path(path).parent))
```

Comment:
Is this just returning the path of the last file? Maybe that was intentional? Should we check the paths are all the same for all files?

Reply:
If we change as discussed above this would become irrelevant.